task_sdk/src/airflow/sdk/api/datamodels/_generated.py

```diff
@@ -18,14 +14,19 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+# generated by datamodel-codegen:
+#   filename:  http://0.0.0.0:9091/execution/openapi.json
+#   version:   0.27.3
```

Comment:
Why this change?

Reply:
Hmmm. not quite sure. it's auto generated

Reply:
updated, thanks!